OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs

```diff
@@ -106,6 +111,11 @@ public MultiplayerLogic(Widget widget, ModData modData, Action onStart, Action o
 			widget.Get<ButtonWidget>("BACK_BUTTON").OnClick = () => { Ui.CloseWindow(); onExit(); };
 			Game.LoadWidget(null, "GLOBALCHAT_PANEL", widget.Get("GLOBALCHAT_ROOT"), new WidgetArgs());
 
+			lanGameLocations = new List<BeaconLocation>();
+			lanGameProbe = new Probe(Server.MasterServerPinger.OpenRALANGame);
+			lanGameProbe.BeaconsUpdated += locations => lanGameLocations = locations;
```

Comment:
> You will need to refresh the server list, too.
This isn't clear to me. You refresh server list after button click in onComplete. At the same time it retrieves server games it appends local games so the window is refreshed for both server and local games.